Project.toml

```
@@ -27,13 +27,13 @@ Wavelets = "29a6e085-ba6d-5f35-a997-948ac2efa89a"
 
 [compat]
 FFTViews = "0.3"
-FFTW = "1.2"
+FFTW = "1.2, 1.3"
```

`"1.2"` means  `[1.2.0, 2.0.0)`, which is a superset of `"1.3"`, i.e., `[1.3.0, 2.0.0)`.

I guess FFTW "1" also works?

```suggestion
FFTW = "1"
```